.github/workflows/build.yml

```diff
@@ -9,10 +9,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - uses: actions/setup-python@v4
         with:
-          python-version: 3.11
+          python-version: 3.12
+          allow-prereleases: true
```

Comment:
I think we can remove this now, right?

Reply:
It will allow me to play with Python 3.11 and will sit dorment until we update line 15.